Formula/go@1.13.rb

```diff
@@ -1,9 +1,9 @@
 class GoAT113 < Formula
   desc "Go programming environment (1.13)"
   homepage "https://golang.org"
-  url "https://dl.google.com/go/go1.13.12.src.tar.gz"
```

Comment:
why not use `https://dl.google.com/go/go1.13.13.src.tar.gz`?

Reply:
Whoops. I copied the other URL from the main download page. Fixing...